### ISSUE TITLE
TypeError: post() missing 1 required positional argument: 'self

### ISSUE THREAD

**Issue by ghost** [sanjeevan/flask-json-schema#2]
I set up as explained in the docs, I am working with flask restful but I keep getting this error when I try to run. TypeError: post() missing 1 required positional argument: 'self'

**Comment by sanjeevan** [comment]
I'll going to need a bit more info to find the issue. Can you provide sample code that replicates the issue?

**Comment by ghost** [comment]
###### So first I initialize it in my init.py file
<img width="466" alt="screen shot 2018-12-20 at 19 08 16" src="https://user-images.githubusercontent.com/16600069/50296217-06a5b880-048b-11e9-9e45-439b53662d7d.png">

##### then I create my validator object in my view.py file 
<img width="601" alt="screen shot 2018-12-20 at 19 08 34" src="https://user-images.githubusercontent.com/16600069/50296289-3c4aa180-048b-11e9-8a68-b7e81a7db64b.png">

##### then I call it on my class based view (flask-restful)
<img width="946" alt="screen shot 2018-12-20 at 19 09 09" src="https://user-images.githubusercontent.com/16600069/50296355-669c5f00-048b-11e9-855a-eb85abde249c.png">

##### but this is the error I get when I call that endpoint
<img width="1139" alt="screen shot 2018-12-20 at 19 09 42" src="https://user-images.githubusercontent.com/16600069/50296458-964b6700-048b-11e9-918d-dbab3c8e4a0a.png">

so basically that's all, and all the decorators(@schema,@app) are properly imported into the file
